src/http.rs

```diff
@@ -974,10 +994,16 @@ fn vrps(
         current, filters, metrics
     );
 
-    Response::builder()
-    .header("Content-Type", format.content_type())
-    .header("content-length", stream.output_len())
-    .body(Body::wrap_stream(stream::iter(
+    let mut builder = Response::builder();
+
+    builder = builder.header("Content-Type", format.content_type())
```

Comment:
You don’t need a new assignment here, this can all be done in one fell swoop.

(As a side note: Can you make sure the line length is not more than 78 characters? I need to document that somewhere as it has become nonstandard by now …)